modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -41,7 +41,7 @@ While the OVN-Kubernetes network plugin implements many of the capabilities pres
 * Egress router pods
 --
 
-* If your cluster or surrounding network uses any part of the `100.64.0.0/16` address range, you must choose another unused IP range by specifying the `v4InternalSubnet` spec under the `spec.defaultNetwork.ovnKubernetesConfig` object definition. OVN-Kubernetes uses the IP range `100.64.0.0/16` internally by default.
+* If your cluster or surrounding network uses any part of the `100.64.0.0/16`, `169.254.169.0/29`, `100.88.0.0/16`, `fd98::/64`, `fd69::/125`, and the `fd97::/64` address ranges, you must choose another unused IP address range by specifying the `v4InternalSubnet` spec under the `spec.defaultNetwork.ovnKubernetesConfig` object definition. OVN-Kubernetes uses the IP address range `100.64.0.0/16` internally by default.
```

Comment:
May be, `OVN-Kubernetes uses these IP address ranges internally by default` ?

Reply:
Yes, it is required and it is mentioned our other DOC which for CIDR ranges : https://docs.openshift.com/rosa/networking/cidr-range-definitions.html

Reply:
This looks fixed now: https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#:~:text=OVN%2DKubernetes%2C%20the,cluster%20or%20infrastructure.